utils/profiles.ts

```diff
@@ -82,6 +83,7 @@ export function retrievedProfileToExtendedProfile(
     last_app_version:
       profile.lastAppVersion !== "UNKNOWN" ? profile.lastAppVersion : undefined,
     preferred_languages: profile.preferredLanguages,
+    reminder_status: profile.reminderStatus,
```

Comment:
Just a consideration. We want to propagate the `UNSET` option to the client? Could be enough an optional `ENABLED | DISABLED` otherwise the app should implement a more complex check on this property like `reminder_status === "UNSET" | reminder_status === undefined` to handle the unset status.
```suggestion
    reminder_status: profile.reminderStatus !== "UNSET" ? profile.reminderStatus : undefined,
```
With this change we hide the complexity to handle the `UNSET` status that will exists only on `fn-app` and `cosmosdb model`.
What do you think about?

Reply:
I think that it's a good catch.